src/client.js

```diff
@@ -1822,23 +1949,31 @@ function SugarApi(args) {
          * States should be used to track conditions or parameters that should be applied to all requests made
          * regardless of their source.
          *
-         * @param {String} key
+         * @param {string} key
          * @param {Mixed} value
```

Comment:
Can we change "Mixed" to "*" since we're changing stuff around it? "Mixed" isn't supported by JSDoc.